src/app/[lang]/layout.tsx

```diff
@@ -21,15 +25,28 @@ export const metadata: Metadata = {
   description: "Ticket flow PoC",
 };
 
-export default function RootLayout({
+const tenant = process.env.TENANT_ID;
+
+export default async function RootLayout({
   children,
+  params,
 }: Readonly<{
   children: React.ReactNode;
+  params: { locale: string; lang: string };
```

Comment:
the types for params should be a Promise (https://nextjs.org/docs/app/building-your-application/routing/dynamic-routes)
```suggestion
  params: Promise<{ locale: string; lang: string }>;
```